src/renderer/views/Config.vue

```diff
@@ -81,32 +80,121 @@
+                                        </x-label>
+
+                                        <x-throbber v-else class="w-8 text-yellow-300"></x-throbber>
+                                    </x-button>
+                                </x-card>
+                            </template>
+                            <template v-else>
+                                <x-label 
+                                    class="text-neutral-400 text-[0.9rem] !pt-0 !mt-0" 
+                                    v-if="usbManager.ptDevices.value.length == 0"
+                                >
+                                    Press the button below to add USB devices to your passthrough list
+                                </x-label>
+                                <TransitionGroup name="devices" tag="x-box" class="flex-col gap-2 mt-4">
+                                    <x-card 
+                                        class="flex justify-between items-center px-2 py-0 m-0 bg-white/5"
+                                        :class="{ 'opacity-75': !usbManager.isPTDeviceConnected(device) }"
+                                        v-for="device, k of usbManager.ptDevices.value" 
+                                        :key="`${device.vendorId}-${device.productId}`"
+                                    >
+                                        <div class="flex flex-row gap-2 items-center"> 
+                                            <Icon v-if="!usbManager.isPTDeviceConnected(device)" class="inline-flex text-red-500 size-7" icon="clarity:warning-solid">
+                                            </Icon>
+                                            <p class="text-base !m-0 text-gray-200">
+                                                {{ usbManager.stringifyPTSerializableDevice(device) }}
+                                            </p>
+                                        </div>
+                                        <x-button @click="removeDevice(device)" class="mt-1 !bg-gradient-to-tl from-red-500/20 to-transparent hover:from-red-500/30 transition !border-0">
+                                            <x-icon href="#remove"></x-icon>
+                                        </x-button>
+                                    </x-card>
+                                </TransitionGroup>
+                                <x-button 
+                                    v-if="availableDevices.length > 0"
+                                    class="mt-4 !bg-gradient-to-tl from-blue-400/20 shadow-md shadow-blue-950/20 to-transparent hover:from-blue-400/30 transition"
+                                    @click="refreshAvailableDevices()"
+                                >
+                                    <x-icon href="#add"></x-icon>
+                                    <x-label>Add Device</x-label>
+                                    <TransitionGroup ref="usbMenu" name="menu" tag="x-menu">
```

Comment:
Restricting the height of the list seems to be enough. It already follows overflow-auto rules if using with a number <100.
<img width="604" height="388" alt="image" src="https://github.com/user-attachments/assets/cc013cd7-8779-473b-9580-5182273d9573" />
 

```suggestion
                                    <TransitionGroup ref="usbMenu" name="menu" tag="x-menu" class="max-h-64">
```

Reply:
Thanks for pointing this out!